### PR TITLE
Feat/permissions flow

### PR DESCRIPTION
**Pull request by NicoMeyer**
## Description
* Make some changes in the front to conditional rendering components and forms according to the role of the user.
* It isn't necessary now to conditional rendering in the subprojectId view, but I left a comment on how to access the role of the user from the project that contains the subproject. I though that this will be useful in the future.

## Requirements
No special requirements

## Additional changes
* Make a ProjectInfo component so the ProjectId page isn't too saturated.

### CODE REVIEW

File: components/organizations/OrganizationInfo.jsx [hunk 1]
```diff
@@ -73,29 +73,34 @@ export default function OrganizationInfo({ organizationData }) {
...
-                sm:px-4 py-1.5 disabled:opacity-30 transition-all duration-150 
-                bg-primary-neutral-hover 
-                text-primary-contrast hover:bg-primary-neutral text-xs sm:text-base"
-            >
-              Add collaborators <UserPlusIcon className="h-5 md:h-6 aspect-square" />
-            </div>
-          </a>
-        </button>
+        {organizationData.organizationPermission.role == "a" ||
+        organizationData.organizationPermission.role == "w" ? (
```
Comment (ignaciomansuy) [inline components/organizations/OrganizationInfo.jsx:77]:
```suggestion
        { ["a", "w"].includes(organizationData.organizationPermission.role) ? (
```

File: components/projects/ProjectInfo.jsx [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+          </div>
+          <div className="pt-1 pb-3">
+            {subprojects && (
+              <>
+                <div className="w-full justify-items-center ">
+                  <div className="">
+                    <div className="flex justify-between items-center py-2">
+                      <p className="text-lg md:text-lg lg:text-2xl pb-2 pl-1">Subprojects</p>
+                      {projectData.projectPermission.role == "a" ||
+                      projectData.projectPermission.role == "w" ? (
```
Comment (ignaciomansuy) [inline components/projects/ProjectInfo.jsx:49]:
```suggestion
                      {["a", "w"].includes(projectData.projectPermission.role) ? (
```

File: components/organizations/OrganizationInfo.jsx (hunk 1 above)
Comment (NicoMeyer) [reply components/organizations/OrganizationInfo.jsx:77]: [].includes no lo soporta Microsoft Edge entonces por eso no lo quise incluir

File: components/projects/ProjectInfo.jsx (hunk 2 above)
Comment (NicoMeyer) [reply components/projects/ProjectInfo.jsx:49]: El mismo motivo que arriba

File: components/organizations/OrganizationInfo.jsx (hunk 1 above)
Comment (ignaciomansuy) [reply components/organizations/OrganizationInfo.jsx:77]: oh crazy shit

**Review by ignaciomansuy — APPROVED** [review]

**Pull request merged**